miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -746,6 +761,52 @@ def set_carpet_cleaning_mode(self, mode: CarpetCleaningMode):
             == "ok"
         )
 
+    @command()
+    def dust_collection_mode(self) -> Optional[DustCollectionMode]:
+        """Get the dust collection mode setting."""
+        self._verify_auto_empty_support()
+        try:
+            return DustCollectionMode(self.send("get_dust_collection_mode")["mode"])
```

Comment:
Just a side note, is there some other information delivered by that call? If yes, it's probably better to separate the request from accessing the property just in case.